app/Http/Controllers/Dashboard.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Helpers\Redirect;
+use App\Http\Middleware\IsAuthenticatedMiddleware;
+use App\Libraries\Controller;
+use App\Libraries\Database;
+
+class Dashboard extends Controller
+{
+
+  public function index()
+  {
+    if (!IsAuthenticatedMiddleware::handle()) Redirect::To('/login');
+
+    $data = [];

```

Comment:
You can set the method to have $data parameter as empty array instead of assigning it to a variable and passing it to the view method.